include/grpc/event_engine/event_engine.h

```diff
@@ -331,8 +336,11 @@ class EventEngine {
   // de-experimentalize this API.
   virtual bool IsWorkerThread() = 0;
 
-  /// Creates and returns an instance of a DNSResolver.
-  virtual std::unique_ptr<DNSResolver> GetDNSResolver() = 0;
+  /// Creates and returns an instance of a DNSResolver, optionally configured by
+  /// the \a options struct.
+  virtual std::unique_ptr<DNSResolver> GetDNSResolver(
+      const DNSResolver::ResolverOptions& options =
```

Comment:
@ctiller tends to really dislike default-value parameters.  Might be better to just make this required; it's not that onerous to require the caller to instantiate an empty struct when they call this method.

Reply:
Done